fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/PlannerExpression.java

```diff
@@ -113,24 +113,12 @@
      * This is needed for graph integration into IntelliJ as IntelliJ only ever evaluates selfish methods. Add this
      * method as a custom renderer for the type {@link PlannerExpression}. During debugging you can then for instance
      * click show() on an instance and enjoy the query graph it represents rendered in your standard browser.
-     *
+     * @param renderSingleGroups -- instruct the explain graph to not render group references with just one member
```

Comment:
Adjusting to [match Javadoc conventions](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#styleguide):
```suggestion
     * @param renderSingleGroups whether to render group references with just one member
```

Reply:
Done.